python/src/Distribution.i

```diff
@@ -199,6 +206,9 @@ class SciPyDistribution(PythonDistribution):
         moment = self._dist.moment(n)
         return [moment]
 
+    def isContinuous(self):
+        return self._dist.__class__.__name__ not in ('rv_frozen', `rv_continuous`, `rv_histogram`)
```

Comment:
```suggestion
        return self._dist.__class__.__name__ in (`rv_continuous_frozen`, `rv_histogram_frozen`)
```
This makes more sense to me. It is not compliant with scipy<1.9, however.

Reply:
Before `1.9` distributions were always of type `rv_frozen`